nemo/collections/asr/parts/submodules/subsampling.py

```diff
@@ -91,48 +93,40 @@ def __init__(self, subsampling, subsampling_factor, feat_in, feat_out, conv_chan
         else:
             raise ValueError(f"Not valid sub-sampling: {subsampling}!")
 
-        in_length = torch.tensor(feat_in, dtype=torch.int)
-        for i in range(self._sampling_num):
-            # length=int(in_length),
-            out_length = calc_length(
-                lengths=in_length,
-                padding=self._padding,
-                kernel_size=self._kernel_size,
-                stride=self._stride,
-                ceil_mode=self._ceil_mode,
-            )
-            in_length = out_length
-
-        out_length = int(out_length)
-        self.out = torch.nn.Linear(conv_channels * out_length, feat_out)
+        self.one = torch.tensor(1, dtype=torch.float)
```

Comment:
Do we need this self.one? Why not use 1.0 or torch.tensor(1, dtype=torch.float) in the place we need explicitly?

Reply:
That was result of some tweaking for ONNX and TS export - let me double check how it renders with current code.